rust/headless-client/src/lib.rs

```diff
@@ -89,9 +90,19 @@ pub enum IpcServerMsg {
     TerminatingGracefully,
 }
 
+/// Messages that go all the way from connlib to the GUI process, or to the headless Client
+#[derive(Debug, serde::Deserialize, serde::Serialize)]
+pub enum ConnlibMsgToGui {
```

Comment:
I'm open to bikeshedding the names here.

I kept `IpcServerMsg` the same. With that `ConnlibMsg` the principle is broadly "messages are named for whence they came" which is maybe not intuitive but it's what I've been using.

Reply:
I propose we inline these into `ConnlibMsg` and `IpcServer`, it involves a bit of duplication but makes thing much easier to read and requires less matching level.s